e2e/definitions/backup/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+resources:
+- ../secrets
+- backup.yaml
+
+namespace: k8up-system
```

Comment:
Maybe we can test the stuff in a different namespace than the one the operator runs in? Like simulating a user namespace.